tests/Http/ClientTest.php

```diff
@@ -231,6 +231,28 @@ public function testClientHasDefaultUserAgent(): void
         $this->assertTrue($client->isHealthy());
     }
 
+    public function testEmptyMasterkeyRemovesAuthHeader(): void
+    {
+        $httpClient = $this->createHttpClientMock(200, '{}');
+        $reqFactory = $this->createMock(RequestFactoryInterface::class);
+        $requestStub = $this->createMock(RequestInterface::class);
+        $requestStub->expects($this->once())
+            ->method('withAddedHeader')
+            ->willReturnCallback(function ($name, $value) use ($requestStub) {
+                $this->assertSame('User-Agent', $name);
+                $this->assertSame(Meilisearch::qualifiedVersion(), $value);
+
+                return $requestStub;
+            });
+        $reqFactory->expects($this->once())
+            ->method('createRequest')
+            ->willReturn($requestStub);
+
+        $client = new \Meilisearch\Client('http://localhost:7070', '', $httpClient, $reqFactory);
```

Comment:
```suggestion
        $client = new Client('http://localhost:7070', '', $httpClient, $reqFactory);
```

Reply:
@norkunas this made the test fail because I specifically was using the `\Meilisearch\Client::class` instead of the `\Meilisearch\Http\Client::class` 😅